.gitignore

```diff
@@ -9,3 +9,4 @@ bin/
 coverage.txt
 go.work
 .idea
+/pkg/handler/collector/git/guac-data-test
```

Comment:
We put that there in case if the test ever fails and we didn't want to include the git commit. The folder automatically gets deleted by the test cases that run.

Reply:
ah - see my follow-up comment on MkTemp